WireDomain/Sources/WireDomain/Repositories/Team/TeamRepository.swift

```diff
@@ -103,8 +103,9 @@ final class TeamRepository: TeamRepositoryProtocol {
             return member
         }
 
-        await userRepository.deleteUserAccount(
-            for: user,
+        try await userRepository.deleteUserAccount(
+            with: userID,
+            domain: nil,
```

Comment:
don't you have to pass the domain?

Reply:
fixed in https://github.com/wireapp/wire-ios/pull/2001/commits/7f25694dccd08ea38100bc39c633e764ffad7298